cmd/provider/list-default-providers.go

```diff
@@ -0,0 +1,82 @@
+package provider
+
+import (
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/loft-sh/devpod/cmd/flags"
+	"github.com/spf13/cobra"
+)
+
+// ListAvailableCmd holds the list cmd flags
+type ListAvailableCmd struct {
+	flags.GlobalFlags
+
+	Output string
+	Used   bool
+}
+
+var httpClient = &http.Client{
+	Transport: &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	},
+}
+
+func getDevpodProviderList() error {
+	req, err := http.NewRequest("GET", "https://api.github.com/users/loft-sh/repos", nil)
+	if err != nil {
+		return err
+	}
+	resp, err := httpClient.Do(req)
+	if err != nil {
+		return err
+	}
+
+	result, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return err
+	}
+
+	var jsonResult []map[string]interface{}
+	err = json.Unmarshal(result, &jsonResult)
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("List of available providers from loft:")
+	for _, v := range jsonResult {
+		if strings.Contains(v["name"].(string), "devpod-provider") {
+			name := strings.Split(v["name"].(string), "-")
+			fmt.Println("\t", name[len(name)-1])
+		}
```

Comment:
What if there will be a provider called "devpod-provider-[cloud]-v2" in the future?

Might be worth _only_ stripping "devpod-provider-" from the name and using the rest.

Reply:
makes sense yes :+1: